tests/PySys/apt_plugin/apt_remove_dense/run.py

```diff
@@ -0,0 +1,25 @@
+import sys
+
+sys.path.append("apt_plugin")
+from environment_apt_plugin import AptPlugin
+
+"""
+Validate apt plugin remove
+
+When we install a package
+When we remove the package
+Then we dont find it in the list of installed packages
+"""
+
+
+class AptPluginRemoveDense(AptPlugin):
```

Comment:
What is the meaning of "dense" here?

Reply:
If I remember correctly, the dense mean we use the environment class that we wrote to avoid code duplication. These are kind of redundant tests, we will remove.

Reply:
So to avoid code duplication, tests have been duplicated ;-).

This is definitely a good move to extent an `AptPlugin` base test, but that should have been done directly in the former tests rather than in copy-pasted ones.

Reply:
Hi, I kept the old ones as reference. I wasn't sure if you are upset when I just replace them. When we all are fine with it we can remove the old, non-dense versions

Reply:
> I wasn't sure if you are upset when I just replace them.

I will certainly not be upset by an improvement and if I disagree with a change I will simply tell it.